framework/include/controls/ControlInterface.h

```diff
@@ -54,29 +38,25 @@ class ControlInterface
 {
```

Comment:
This is an interface to `ControllableParameter`s... it should probably be `ControllableInterface`.  We just need to be consistent.  That matches `Coupleable` for instance.